### PR TITLE
Delete unused streaming_combinations.jl

### PR DESCRIPTION
**Pull request by lucifer1004**

### CODE REVIEW

File: test/streaming_combinations.jl [hunk 1]
```diff
@@ -92,7 +92,7 @@ end
 send_small_chunk(http, 10)
 send_large_chunk(http, 1000)
 
-send_chunk(http, size)
+send_chunk(http, size) = begin
```
Comment (fredrikekre) [inline test/streaming_combinations.jl:95]:
```suggestion
function send_chunk(http, size)
```

**Review by fredrikekre — COMMENTED** [review]
Is this file not executed in the tests?

**Comment by lucifer1004** [comment]
> Is this file not executed in the tests?

It is certainly not executed. And it seems to be an ancient draft and has not been updated for years. Can we just delete it?

**Comment by quinnj** [comment]
Yeah, I would say we just delete it.

**Comment by lucifer1004** [comment]
So it is deleted.

**Pull request merged**